tests/labhub_test.py

```diff
@@ -294,3 +294,10 @@ def test_invite_me(self):
                               'We\'ve just sent you an invite')
         with self.assertRaises(queue.Empty):
             testbot.pop_message()
+
+
+        testbot.assertCommand('!hey there invite me',
+                              'Command \"hey\" / \"hey there\" not found.',
+                              timeout=100)
```

Comment:
This is unnecessary

Reply:
you asked to move below #298, otherwise, I also thought it was unnecessary

just doin' it right now

Reply:
I mean, `timeout=100` is not necessary

Reply:
sorry, I should have got it... yeah timeout was unnecessary... silly me 😅